### DISCUSSION TITLE
Install pgcli

### DISCUSSION THREAD

**Discussion by NouhJama** [dbcli/pgcli#1439] [Q&A]
I was creating posgres sql database pipeline in a docker container. I set up everythin and worked well untill i wanted to connect my posgres db with pgcli. I used the pip package manager to install it, but it didn't work and pops up an error. So please could you guys helm me with this.
![pgclierror](https://github.com/dbcli/pgcli/assets/143721840/4394003c-7e38-4428-bc7b-7e830163aa17)

**Comment by dbaty** [comment]
Hello. Could you clarify which command you tried and the OS and Python version? The traceback suggests Python 3.12 on Windows, but the error is odd. If the erroring command is `pip install pgcli`, could you please check whether you have the same error with another package (for example `pip install django`)?

**Comment by dbaty** [comment]
No news for 4 months, I am closing this. Feel free to reopen with more details if you still have this issue.
